sdk/resourcemanager/Azure.ResourceManager.Core/src/Generated/ResourceGroupOperations.cs

```diff
@@ -70,11 +67,11 @@ protected ResourceGroupOperations(ResourceOperationsBase options, ResourceGroupR
         /// <inheritdoc/>
         protected override ResourceType ValidResourceType => ResourceType;
 
-        private ResourceGroupsOperations Operations => new ResourcesManagementClient(
-            BaseUri,
+        private ResourceGroupsRestOperations RestClient => new ResourceGroupsRestOperations(
```

Comment:
I see a few missing methods that were in the old operations class that arent here like update and export template,